CONTRIBUTING.md

```diff
@@ -47,8 +47,8 @@ Rake can be found at http://docs.seattlerb.org/rake/.
 
 To run a single test suite
 
-   rake test TEST=path/to/test.rb
+`rake test TEST=path/to/test.rb`
```

Comment:
The `$` makes it clear it's a shell command. Plus, nobody should blindly copy-paste shell commands.